src/reusableState/customList/StringListActions.ts

```diff
@@ -0,0 +1,34 @@
+import {IReduxAction} from '../../utils/ReduxUtils';
+
+export const StringListActions = {
+    add: 'ADD_STRING_LIST',
+    remove: 'REMOVE_STRING_LIST',
+    addValue: 'ADD_VALUE_STRING_LIST',
+    removeValue: 'REMOVE_VALUE_STRING_LIST',
+};
+
+export interface IStringListPayload {
+    id: string;
+    list?: string[];
+    value?: string;
+}
+
+export const addStringList = (id: string, list?: string[]): IReduxAction<IStringListPayload> => ({

```

Comment:
`(id: string, list: string[] = [])`  will allow you to remove your default array here https://github.com/coveo/react-vapor/pull/863/files#diff-b1aec608a3062febfd6d1d9a3f6d70e3R34 I think it would be cleaner